src/lib.rs

```diff
@@ -22,6 +22,8 @@
 //! - `iota_streams_ddml`: Modules for working with
 //! the IOTA data description language called DDML, in which all Streams messages are encoded
 
+// CORRECTNESS: Channel is not the high-level API of Streams
```

Comment:
Both the Streams crate module docs and the crate organization suggests that `channels` is the high-level API of Streams, and `core` is the low-level. However, the very same crate summary is `A cryptographic framework for building secure messaging applications`, which `channels` definitely isn't; `channels` is one implementation of the `streams` framework. This is specially relevant since corporate-wise, `Streams` is being marketed as what `channels` is, not the cryptographic messaging framework. These naming contractions lead to confusion and misunderstandings and ultimately, frustration.

Besides docs, which are easily rewriten, the important issue here is the crate structure. There should be an univocal client => library